politeiawww/wsdcrdata.go

```diff
@@ -26,13 +30,46 @@ var (
 	// errSubNotFound is emitted when attempting to unsubscribe to an
 	// event that has not yet been subscribed to.
 	errSubNotFound = errors.New("subscription not found")
+
+	errShutdown = errors.New("ws connection is shut down")
```

Comment:
Needs a comment.

This is the second time I've had to leave this same comment in this PR (prev was for newBlockSub).  This is sloppy.  Step it up and review your code more carefully.

Reply:
Sorry about that I'll pay more attention.